api/mesh/v1alpha1/mesh.proto

```diff
@@ -124,11 +124,8 @@ message TracingBackend {
 }
 
 message DatadogTracingBackendConfig {
-  // Address of datadog collector.
+  // Address of datadog collector. HOST:IP
```

Comment:
Another way to address forward compatibility could be to use a `oneof` with the address an port in one branch. You could fill in another branch later.

Reply:
Good point. I think that would be appropriate if we were worried about disparate, unforeseeable types, but this is really just changing it to what we decided upon in a subsequent PR in order to avoid unnecessary churn. I don't think we need to totally future-proof it.